openedx/core/djangoapps/catalog/utils.py

```diff
@@ -66,6 +72,19 @@ def get_programs(uuid=None, type=None):  # pylint: disable=redefined-builtin
         return []
 
 
+def get_program_type(name):
```

Comment:
Why make another API call when we already have the small list of program types cached?